src/phones/controllers/phones.controller.ts

```diff
@@ -58,31 +61,33 @@ export class PhonesController {
 
   @Get(':id')
   @ApiOperation({ summary: 'Get phone by id' })
-  @ApiResponse({ status: 200, description: 'The phone found', type: Phone })
-  @ApiQuery({
+  @ApiOkResponse({ status: 200, description: 'Ok', type: Phone })
```

Comment:
En el caso de 200, la respuesta no es un objeto con message Ok, sino que trae un telefono.  ¿No sería mejor el `'The phone found'` como estaba antes?